api/files/views.py

```diff
@@ -171,3 +175,33 @@ def get_serializer_context(self):
         context = JSONAPIBaseView.get_serializer_context(self)
         context['file'] = self.file
         return context
+
+
+class FileCedarMetadataRecordsList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin, FileMixin):
+
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        PermissionWithGetter(ContributorOrPublic, 'target'),
+    )
+    required_read_scopes = [CoreScopes.CEDAR_METADATA_RECORD_READ]
+    required_write_scopes = [CoreScopes.NULL]
+
+    serializer_class = CedarMetadataRecordsListSerializer
+
+    view_category = 'files'
+    view_name = 'file-cedar-metadata-records-list'
+
+    def get_default_queryset(self):
+        self.get_file()
+        obj = self.get_file()
```

Comment:
```suggestion
        obj = self.get_file()
```
The first call is unnecessary